tina/config.ts

```diff
@@ -187,6 +187,65 @@ export default defineConfig({
                   },
                 ],
               },
+              {
+                type: "object",
+                label: "Quote Section",
+                name: "quoteSection",
+                fields: [
+                  {
+                    type: "object",
+                    label: "Request Quote Form",
+                    name: "requestQuoteForm",
+                    fields: [
+                      {
+                        type: "string",
+                        label: "Form Title",
+                        name: "formTitle",
+                      },
+                      {
+                        type: "string",
+                        label: "Form Field 1 Placeholder",
+                        name: "field1Placeholder",
+                      },
+                      {
+                        type: "string",
+                        label: "Form Field 2 Placeholder",
+                        name: "field2Placeholder",
+                      },
+                      {
+                        type: "rich-text",
+                        label: "Form Field 3 Placeholder",
+                        name: "field3Placeholder",
+                      },
```

Comment:
I intentionally left that field out of the CMS because if there isn't an email entered the form submit won't work and I didn't want the client to accidentally change that field

Reply:
Ahh I remember now that I was going to ask your opinions on this - I started connecting the form to TinaCMS but then stopped (you can see the `field3Placeholder` and `submitButtonText` props aren't used). I realized that if we let the client modify any of these fields, not just email, won't that completely mess with the form submit logic/state setting etc. that we need to do? I wasn't exactly sure how it would play out so I was planning on leaving the CMS connection in and when we work on the ticket to set up submit functionality we can adjust accordingly.